queue.tf

```diff
@@ -2,10 +2,10 @@ locals {
   # https://docs.aws.amazon.com/lambda/latest/dg/with-sqs.html#events-sqs-queueconfig
   # To allow your function time to process each batch of records, set the source queue's visibility timeout to at least 6 times the timeout that you configure on your function. The extra time allows for Lambda to retry if your function execution is throttled while your function is processing a previous batch.
   visibility_timeout_seconds = coalesce(
-    # Use whatever the user provided if it is not null
```

Comment:
The level of indentation for arguments of the function shouldn't be the same as of the function itself. That hurts readability.